frontend/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -129,17 +136,17 @@ export default ({
   computed: {
     ...mapGetters(['userDisplayNameFromID']),
     interactiveMessage () {
-      const { variant, creator } = this.proposal
-      const baseOptions = { from: this.userDisplayNameFromID(creator) }
+      const { status, creatorID } = this.proposal
+      const baseOptions = { from: `@${creatorID}` }
```

Comment:
Should you use that special variable/constant you created instead of `@` simple literal here?